native/cocos/platform/android/java/src/com/cocos/lib/CocosWebView.java

```diff
@@ -90,7 +95,42 @@ public CocosWebView(Context context, int viewTag) {
          }
 
          this.setWebViewClient(new Cocos2dxWebViewClient());
-         this.setWebChromeClient(new WebChromeClient());
+         this.setWebChromeClient(new WebChromeClient() {
+            // For Android < 3.0
```

Comment:
We don`t need to support API level lower than anroid5.0.

Android 6.0 (API 23), which brought another significant change: dynamic permission management. This update impacted how apps handle permission-related callbacks, such as onPermissionRequest and onGeolocationPermissionsShowPrompt, allowing users to grant or deny permissions in a more granular way, rather than at install time. This shift gave users more control over their data and privacy.